config/routes.rb

```diff
@@ -152,7 +152,11 @@
         only: %i[create destroy],
         constraints: { format: :js },
         defaults: { format: :js }
-      resources :versions, only: %i[show index]
+      resources :versions, only: %i[show index] do
+        get '/dependencies', to: 'versions#dependencies_show'
+      end
+      get "/find-by-name", to:  "versions#transitive_dependencies_show", constraints: { format: :js }
```

Comment:
please use a better name for this route. `find-by-name` sounds more like a function name than a route.